doc/readme.md

```diff
@@ -337,6 +337,18 @@ There are a number of ways to specify a plugin:
 | commit-ish  | Commit/tag/branch reference. If none is specified, 'master' is used
 | subdir      | Sub-directory to find plugin.xml for the specified plugin.
 
+### Algorithm for choosing which plugin version to fetch from npm
+
+When adding a plugin to a project from npm, the CLI will choose which version
+of the plugin to fetch based on the following criteria (listed in order of
+precedence):
+
+1. The plugin version specified in the command (e.g. `cordova plugin add plugin@version`)
+2. The plugin version saved in config.xml (i.e. if the plugin was previously added with `--save`)
+3. The latest plugin version that the current project can support (only applies to plugins that list their [Cordova dependencies] in their `package.json`)
+4. The plugin version pinned by your installed CLI version (only applies to [Apache-developed plugins])
```

Comment:
This will likely need an update in a couple of days - when you remove plugin pinning.